internal/cfg/cfg.go

```diff
@@ -68,6 +68,8 @@ type Wall struct {
 	// Preview percentage to show instances at.
 	ShowAt int `toml:"show_at"`
 
+	WallWindow string `toml:"wall_window"` // Name of the wall window 
```

Comment:
There's trailing whitespace on this line. Running `go fmt ./...` should get rid of it.